### ISSUE TITLE
Recaptcha v2 stopped working?

### ISSUE THREAD

**Issue by PixedBo** [joomla/joomla-cms#42675]
Hello everyone, I don't know whether this issue really concerns Joomla or is something to do with Recaptcha. For the last few days most of my sites that use recaptcha v2 (the checkbox with the tick) have stopped working: the wheel spins endlessly.
I have encountered this on practically all the Joomla3 sites I manage (also because the sites made with Joomla4 almost all use the invisible recaptcha which seems to work).
The strange thing is that on some sites that use external plugins (like RsForm) recaptcha v2 works correctly, while the integrated core recaptcha doesn't work.

Basically: is this just happening to me or is there a problem somewhere on the Joomla and/or recaptcha side?

Thanks!
